### ISSUE TITLE
Failed to stop component `tikv`: signal: interrupt

### ISSUE THREAD

**Issue by zz-jason** [pingcap-incubator/tiup#133]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do?

start a playground, then `ctrl + c`:

```
CLUSTER START SUCCESSFULLY, Enjoy it ^-^
To connect TiDB: mysql --host 127.0.0.1 --port 4000 -u root
Cluster tag: RvpOaaO, you can restore this cluster via: tiup -T RvpOaaO playground
To view the dashboard: http://127.0.0.1:2379/dashboard
^CGot signal interrupt (Component: tidb ; PID: 2082)
Got signal interrupt (Component: tikv ; PID: 2078)
Got signal interrupt (Component: pd ; PID: 2076)
Got signal interrupt (Component: playground ; PID: 2055)
Failed to stop component `playground`: signal: interrupt
[jianzhang.zj@localhost ~]$ Failed to stop component `tidb`: signal: killed
Failed to stop component `tikv`: signal: interrupt
Failed to stop component `pd`: exit status 1
```

2. What did you expect to see?

all components stopped, no annoying `Failed to stop xxx` message.

3. What did you see instead?

as showed in the former section

4. What version of TiUP are you using (`tiup --version`)?

```
[jianzhang.zj@localhost ~]$ tiup --version
tiup version v0.0.4 tiup
Go Version: go1.13
Git Branch: master
GitHash: f74f86a
Build Time: 2020-04-10 15:01:40 +0800
```

**Comment by lonng** [comment]
@HunDunDM PTAL

**Comment by HunDunDM** [comment]
I have solved this problem through #127, and the currently downloaded playground has not been updated to include this PR.
